action.yml

```
@@ -72,6 +75,10 @@ runs:
         source "${{ github.action_path }}/setup/venv.bash"
 
         python "${{ github.action_path }}/action.py" "${{ inputs.inputs }}"
+
+        if [[ ${{ inputs.summary }} ]]; then
```

Nitpick (using `${{ }}` would allow unintended injection here):

```suggestion
        if [[ "${GHA_PIP_AUDIT_SUMMARY}" = "true" ]]; then
```